theano/gof/op.py

```diff
@@ -966,7 +966,7 @@ def make_thunk(self, node, storage_map, compute_map, no_recycling):
         if new_node is not None:
             node = new_node
 
-        if self._op_use_c_code:
```

Comment:
I don't think that is a good idea, we should fix the get/setstate of the Op in question instead.
Which Op was that?